packages/EasyApiToken/src/Factories/Decoders/AbstractJwtTokenDecoderFactory.php

```diff
@@ -96,12 +97,19 @@ protected function createJwtDriver(string $driver, array $options): JwtDriverInt
      */
     private function createAuth0Driver(array $options): Auth0JwtDriver
     {
+        $cache = null;
+        if (array_key_exists('cache_path', $options) && empty($options['cache_path']) === false) {
```

Comment:
This can be written as `if (empty($options['cache_path'] ?? null)) === false) {`.

Reply:
fixed